packages/components/src/components/link/link.scss

```
@@ -0,0 +1,90 @@
+@charset "utf-8";
+
+@use "@db-ui/foundations/build/scss/variables" as *;
+@use "@db-ui/foundations/build/scss/scaling-placeholder" as *;
+@use "@db-ui/foundations/build/scss/color-placeholder" as *;
+@use "@db-ui/foundations/build/scss/helpers/functions" as *;
+@use "@db-ui/foundations/build/scss/icon/icons.helpers" as *;
+
+.db-link {
+	color: var(--db-current-color, $db-colors-neutral-on-bg-enabled);
+	display: flex;
+	border-radius: to-rem($pxValue: 6);
+
+	// Link underline
+	// TODO: We should evaluate whether we could move this declaration to the @font-face declarations even already
+	text-underline-position: from-font;
+	text-decoration-thickness: to-rem($pxValue: 1);
+
+	// Changing color on hover should be animated by a transition
+	transition: outline 0.06s, color $db-transition-color;
+
+	&:hover {
+		color: inherit;
+	}
+
+	// "disabled" links
+	&[aria-disabled="true"] {
+		opacity: var(--db-disabled-text-opacity, 0.5);
```

```suggestion
		opacity: var(--db-disabled-text-opacity, $db-disabled-text-opacity);
```

There should most likely be a SCSS variable `$db-disabled-text-opacity` that we could use instead.

---

@mfranzke why do we need both? This is not a design token.